specification/src/specification/queries/sorting.md

```diff
@@ -16,6 +16,11 @@ To compute the ordering from the `order_by` field, data connectors should implem
 
 ### Type `column`
 
+The property `element.target.name` refers to a column name.
```

Comment:
I think this still needs to be moved to its own subsection of one or the other page, and linked from the other, please.

Reply:
I have done this. I'm not sure it really improves the readability of the spec though.

Reply:
I do think it's useful to have a fragment that we can link to.